src/apps/Permissions/AppCard.js

```diff
@@ -16,10 +15,8 @@ class AppCard extends React.PureComponent {
     const instanceTitle = `Address: ${proxyAddress}`
     return (
       <Main onClick={this.handleClick}>
-        <Icon>
-          <RemoteIcon size={28} src={appIconUrl(app)} />
-        </Icon>
-        <Name>{name}</Name>
+        <AppIconCard isCoreApp={name === 'ACL'} app={app} size={28} />
```

Comment:
What do you think about just moving `isCoreApp` to `AppIconCard`, as I imagine we'll have more apps to add (e.g. the `EVMScriptsRegistry`). We should also use the app id to detect this, rather than the name.